README.md

```diff
@@ -103,7 +103,29 @@ Workmanager().registerOneOffTask(
 );
 ```
 
-Tasks registered this way will appear in the callback dispatcher using as `Workmanager.iOSBackgroundProcessingTask`.
+**NOTE**
```

Comment:
Please add a line or two to explain the purpose of registering the tasks and make a note that this MUST be done in AppDelegate and Info.plist in order to work.